research/templates/research/browse.html

```diff
@@ -347,11 +347,11 @@ <h2 class="pt-3"></h2>
                         <div class="btn-group me-2" role="group">
                             <button type="button" class="btn btn-outline-secondary metadata-form" data-path="" title="Open metadata form">Metadata</button>
                         </div>
-                        <div class="btn-group me-2" role="group">
+                        <div class="btn-group me-2" role="group" title="Cannot create files in folder with apostrophe in name" data-toggle="tooltip" tabindex="0">

```

Comment:
I'm not sure if I should try to make this more accessible by removing these when the tooltip is not visible.